api/metadata/formatters.py

```diff
@@ -0,0 +1,141 @@
+from website import settings
+from website.identifiers.metadata import (
+    datacite_format_creators_json,
+    datacite_format_subjects_json,
+)
+
+class FileMetadataFormatter(object):
+
+    @property
+    def metaschema_id(self):
+        raise NotImplementedError('FileMetadataFormatter subclasses must define metaschema_id.')
+
+    def validate(self, metadata):
```

Comment:
This will be implemented here (on the base class) because `validate()` should just called jsonschema's `validate()`